### DISCUSSION TITLE
Useful Commands

### DISCUSSION THREAD

**Discussion by gcsrellis** [Security-Onion-Solutions/securityonion#1985] [Unsupported Versions] [answered]
Is there a list of useful commands on a 1 page document like version 16 had.  I've setup Security Onion 2 next to version 16 and I'm trying to learn the differences so we can migrate.  Simple commands like so-setup are no longer?

**Comment by dougburks** [comment, marked as answer]
You can find a link to the Security Onion 2.3 Cheat Sheet in the upper right corner in SOC and also in our online documentation:
https://docs.securityonion.net/en/2.3/cheat-sheet.html

**Reply by gcsrellis** [reply]
Thanks Doug, I found it right after posting.

**Reply by gcsrellis** [reply]
> You can find a link to the Security Onion 2.3 Cheat Sheet in the upper right corner in SOC and also in our online documentation:
> https://docs.securityonion.net/en/2.3/cheat-sheet.html

For testing purposes, what is the recommended way to re-run setup once a node is configured?  I dont see a script that would do this.

**Reply by dougburks** [reply]
This discussion has already been marked as answered. If you have further questions or problems, please start a new Discussion. Thanks!
